Tutorial/WindowsVersion/tutorial.md

```diff
@@ -0,0 +1,39 @@
+1. Download SPTAG.
```

Comment:
Before starting the step by step guide, may you introduce what's this tutorial is? what's the scenario it will support?

Reply:
Done. Introduced a new paragraph.